community-membership.md

```diff
@@ -8,6 +8,7 @@ scoped by these sub-areas.
 
 | Role | Responsibilities | Requirements | Defined by |
 | -----| ---------------- | ------------ | -------|
+| outside collaborator | new active contributor to the community | sponsored by 1 member.  existing contributions to the project. | Kubernetes GitHub org member. |
```

Comment:
defined by: read-only collaborator on a repository in the kubernetes org

How many people do we have that are collaborators vs. members? 

My concern with putting this in the table is it might imply to someone that this is a required step in the ladder.  I would much prefer people just jump straight to member, than think they have to go through this first.  Managing collaborators is noisier.